waggle-dance-core/pom.xml

```diff
@@ -285,6 +286,16 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-maven-plugin.version}</version>
         <configuration>
+          <requiresUnpack>
+            <!-- We are hitting an issue introduced in hive-common-2.1.1 and fixed in https://issues.apache.org/jira/browse/HIVE-17155
```

Comment:
Ideally we'd remove this later as it's extra code here which we shouldn't need. So I think it wouldn't hurt saying that but fine.